Scripts/Characters/Player/Player.cs

```diff
@@ -11,9 +11,11 @@ public partial class Player : CharacterBody2D
     public const float JumpVelocity = -400.0f;
     
     // Powers
+    public bool DoubleJumpEnabled = false;
```

Comment:
In a perfect world, we'd have these as components that were attached and detached from the player as we needed instead of flags.  This is not that world, but on our next model it's something we want to consider.  

You can *try* to tackle it if you'd like, basically you'd need something like:
Powerup.cs

- public string InputMapName;
- public string DisplayName;
- public abstract void Execute()

Dash : Powerup

public void Execute()
{
   // Logic for dashing
}


etc... But not important now, you'd also need a listener on _PhysicsProcess in player listening for input and checking it against the input map names of the powers the player has